pkg/asset/manifests/network.go

```diff
@@ -208,18 +195,48 @@ func (no *Networking) GenerateCustomNetworkConfigMTU(ic *installconfig.InstallCo
 		return nil, nil
```

Comment:
If there is no customMTU, we don't need the remaining logic?

Reply:
Right. This is preserved from @mtulio's original code:

https://github.com/openshift/installer/blob/a04f578f3f41cfedf0f847c59d487c49e55347c1/pkg/asset/manifests/network.go#L207-L209

But that's a flag that indicates the various conditions for which we would need to generate a cno manifest with custom mtu. If it's false we're fine with the default and don't need the manifest.

Reply:
Correct, overall we want custom MTU config when, at least, one of the following config is set:
- user defined on install-config
- the edge compute pool is defined - we lowered default to the lowest supported value by AWS to support default deployment across all AWS edge zones (user-defined has precedence)